File: 2418-sort-the-people/2418-sort-the-people.py
class Solution:
    def sortPeople(self, names: List[str], heights: List[int]) -> List[str]:
        a = []
        for i in range(len(heights)):
            a.append([ heights[i], names[i]])
        
        a.sort(reverse = True)
        
        b = []
        for i in range(len(a)):
            b.append(a[i][1])
        return b

File: 2418-sort-the-people/2418-sort-the-people.js
/**
 * @param {string[]} names
 * @param {number[]} heights
 * @return {string[]}
 */
var sortPeople = function(names, heights) {
    let a1 = []
    for (let i = 0; i<names.length; i++){
        a1.push([heights[i], names[i]])
    }
    a1.sort((a,b) => b[0] -a[0])
    b = []
    for( i = 0; i<a1.length; i++){
        b.push(a1[i][1])
    }
    return b
    
    
};